cpp/server/event.cc

```diff
@@ -183,7 +183,7 @@ bool EventLoop::EraseCheck(std::deque<FD*>::iterator* pfd) {
 // static
 void EventLoop::Set(int fd, fd_set* fdset, int* max) {
   DCHECK_GE(fd, 0);
-  CHECK_LT((unsigned)fd, FD_SETSIZE);
+  CHECK_LT((unsigned)fd, (unsigned)FD_SETSIZE);
```

Comment:
There's no doc I've seen that says FD_SETSIZE must be unsigned. If it could be signed on some platforms (which makes no real sense) then it won't compile without the casts.